code/modules/shuttle/shuttle.dm

```diff
@@ -21,8 +21,6 @@
 	var/dwidth = 0	//position relative to covered area, perpendicular to dir
 	var/dheight = 0	//position relative to covered area, parallel to dir
 
-	shuttle_abstract_movable = 1
```

Comment:
Which now that I think about it, it should be the other way around, I forget if I told you to add it to all docks or only mobile docks during the rebase but you aparently added it to both.